app/models/account.js

```diff
@@ -1,10 +1,11 @@
 import Model, { attr, belongsTo } from '@ember-data/model';
-import { alias } from '@ember/object/computed';
 
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes
-export default Model.extend({
-  user: belongsTo('person'),
-  gebruiker: alias('user'),
-  voId: attr('string'),
-});
+export default class Account extends Model {
+  @attr('string') voId;
+
+  @belongsTo('person') user;
+
+  get gebruiker() {
```

Comment:
Can you verify whether this property is actually used? If so, can it be replaced with `user`?

Reply:
Yeah, it's used in the `@lblod/ember-mock-login` package here: https://github.com/lblod/ember-mock-login/blob/3aafb710617f63783343ac32eff29ddaf19b67a0/addon/components/login/each-account.hbs#L4

That's the only place though, perhaps I can add a comment and when/if that gets replaced in the package we can remove this getter?

Reply:
Comment is probably a good idea :+1:

Reply:
Will add the comment. Looking at this, shouldn't user be a relationship with `user` instead of `person`? Looking at the resource definition I think that's the case. Only noticed because I was locally trying the upgrade to 3.28 and at that version ember complains about the mismatch.

Reply:
Yes, should indeed be `user` looking at mu-cl-resources config :+1:  
I was a bit confused how the mock-login used to work then? :confused: I assume Ember is currently casting the `user` resource it gets from the backend in a `person` ember-data record? And since they both have `firstName` and `lastName` properties we didn't notice the mismatch until now...

Reply:
this model was either wrong, or we don't know the reason for this approach. If it was just for mock-login then maybe it's fine with the other changes.
I'm not fully convinced that everything will work perfectly, but I couldn't tell you why it was this way if there was any other reason.

Move fast and break things (after custom jenkins build)